tasks/04_pages/test_db.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+from common import db_config, sql
+
+
+if __name__ == '__main__':
+    """Confirm database meets expectations."""
+
+    cases_query = 'SELECT COUNT(*) c from cases;'
+    
+    if db_config.db_type == 'sqlite':
```

Comment:
I wasn't sure which strategy for restarting the task would be more effective. My mental model was to just rerun `make`, but I can see how his might lead to trouble: in cases where a `pages` table had been populated in a prior run of the full project, but then wasn't cleaned or reset when beginning anew. 

A case I was concerned about was when the task had to be restarted, but the sql service hadn't been restarted, which is why I thought it might be worth running the test_db. But `task .py` won't continue if it's unable to connect to the db anyways.

I'll alter this and update the task's `README.md`.